src/Components/TeamCard.jsx

```diff
@@ -2,49 +2,49 @@ import {
   Avatar,
   Card,
   makeStyles,
-  Typography,
-  CardContent,
   IconButton,
-  CardActions,
   CardMedia,
+  List,
+  ListItem,
+  ListItemText,
 } from "@material-ui/core";
 import { LinkedIn } from "@material-ui/icons";
 
 const useStyles = makeStyles({
   card: {
-    width: 200,
     borderRadius: 0,
   },
   name: {
     fontWeight: "bold",
   },
   avatar: {
     width: "100%",
-    height: 200,
+    height: 250,
   },
 });
 
 const TeamCard = (props) => {
   const classes = useStyles();
+  const { firstName, lastName, role, linkedIn } = props.member;
 
-  const {firstName, lastName, role, photo, linkedIn} = props.member;
 
   return (
     <Card className={classes.card}>
       <CardMedia>
-        <Avatar className={classes.avatar} variant="square" src={photo} />
+        <Avatar
+          className={classes.avatar}
+          variant="square"
+          src={`https://cvp-team-photos.s3.us-west-2.amazonaws.com/${firstName+lastName}.jpg`}
```

Comment:
Nice! We should add documentation somewhere that specifies how new team member photos need to be named when being added to the s3 bucket.

Reply:
I guess we could have it in the AWS document that dylan created, unless we want to create another.

Reply:
Something like [this](https://docs.google.com/document/d/19M9ErJop-09XDQHRmEWhI6EZX3k-2iSrlvUiSgK5OnY/edit)

Reply:
Yeah, that's perfect